src/config.rs

```diff
@@ -252,6 +265,7 @@ pub fn construct_config(pc: ProxyConfig) -> Result<Config, Error> {
 
         enable_original_source: parse(ENABLE_ORIG_SRC)?,
         proxy_args: parse_args(),
+        enable_impersonated_identity: parse_default(ENABLE_IMPERSONATED_IDENTITY, true)?,
```

Comment:
```suggestion
        enable_impersonated_identity: parse_default(ENABLE_IMPERSONATED_IDENTITY, false)?,
```

Default on this should be `false` I think - 

We don't want to impersonate by default, and we only need to for some subset environments (serverless).

Reply:
I think you have it backwards? Impersonation is the current behavior, ztunnel gets certs for spiffe://pod-on-my-node. With false it will get one for spiffe://ztunnel (but you would probably run ztunnel itself as a non-ztunnel SA). Basically like a sidecar

Reply:
Yeah - nevermind then, had it backwards.